specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/examples/HostPools_Update_Post.json

```diff
@@ -26,6 +25,26 @@
     }
   },
   "responses": {
+    "200": {
+      "body": {
+        "warnings": [
```

Comment:
For 200, why do we need warnings and errors?

Reply:
its a validation call. the validation call was a good request but there were errors in the validation